src/workers/cdc/primary.ts

```diff
@@ -300,6 +301,13 @@ const onUserChange = async (
         }
       }
     }
+    if (data.payload.after.reputation > data.payload.before.reputation) {
```

Comment:
Nonblocking but I think we should do the check of the threshold here rather than the worker. This is to avoid sending a pubsub event in every reputation increase which is not necessary on some cases.

Reply:
This is now notification for reputation increase, so not directly related to the devcard threshold